lib/sycamore/sycamore/utils/cache.py

```diff
@@ -1,12 +1,15 @@
+from __future__ import annotations
 import hashlib
 import json
 import time
-from typing import Any
+from typing import Any, Optional
 
 import boto3
 import diskcache
 from botocore.exceptions import ClientError
 
+BLOCK_SIZE = 1000000  # 1 MB
```

Comment:
I should have said 1MiB.  That would be 1048576.  We want a clean power of two because disk sectors are 512 bytes, memory pages are 4096 bytes, and the larger structures are built upon them.